public/examples/card-on-file-payment-with-verification.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link href="/app.css" rel="stylesheet" />
+    <script
+      type="text/javascript"
+      src="https://sandbox.web.squarecdn.com/v1/square.js"
+    ></script>
+    <script>
+      const appId = '{APPLICATION_ID}';
+      const locationId = '{LOCATION_ID}';
+
+      async function createCardOnFilePayment(
```

Comment:
when I see this name it makes me thinking about creating a card in file not a payment. Maybe this is the naming we use but how about `createPaymentWithCardOnFile`